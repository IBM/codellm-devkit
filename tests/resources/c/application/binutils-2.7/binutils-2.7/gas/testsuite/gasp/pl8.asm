
	.ALTERNATE
	SDATA	%1+2+3 
	SDATA	"5"



	MACRO	foo
	SDATA	"HI"	!! this will go
	SDATA	"THERE	! this will stay
	ENDM

	foo


	SDATA	<!<this is <a wacky> example!>!!>
	SDATA	"<this is <a wacky> example>!"
	END
