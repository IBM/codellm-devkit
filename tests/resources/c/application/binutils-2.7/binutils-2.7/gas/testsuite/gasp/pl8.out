!

!	.ALTERNATE
!	SDATA	%1+2+3 
	.byte	37,49,43,50,43,51,32
!	SDATA	"5"
	.byte	53
!

!

!

!	MACRO	foo
!	SDATA	"HI"	!! this will go
!	SDATA	"THERE	! this will stay
!	ENDM
!

!	foo
!	SDATA	"HI"	
	.byte	72,73
!	SDATA	"THERE	! this will stay
	.byte	84,72,69,82,69,9,32,116,104,105,115,32,119,105,108,108,32,115,116,97,121
!

!

!	SDATA	<!<this is <a wacky> example!>!!>
	.byte	60,116,104,105,115,32,105,115,32,60,97,32,119,97,99,107,121,62,32,101,120,97,109,112,108,101,62,33
!	SDATA	"<this is <a wacky> example>!"
	.byte	60,116,104,105,115,32,105,115,32,60,97,32,119,97,99,107,121,62,32,101,120,97,109,112,108,101,62,34
!	END
